src/platform/Tizen/ChipDeviceScanner.cpp

```diff
@@ -239,7 +239,10 @@ CHIP_ERROR ChipDeviceScanner::StopChipScan(void)
     ReturnErrorCodeIf(!mIsScanning, CHIP_ERROR_INCORRECT_STATE);
 
     ret = bt_adapter_le_stop_scan();
-    ChipLogError(DeviceLayer, "bt_adapter_le_stop_scan() ret: %d", ret);
```

Comment:
Hopefully there are no automated tests that rely on this print to verify BLE scan success...

Reply:
I believe tests should generally not rely on logs unless we explicitly mark a log for usage for testing (I believe for chip tool we have some bits that need to be somewhat stable for manual tests).

Reply:
There are no automated tests for Tizen platform, at least not in this project repo. I will try to add such tests in the future (using QEMU for running Tizen). Currently I'm trying to tests everything manually using RPi4.